vcpkg-overlays/ports/libsystemd/portfile.cmake

```diff
@@ -0,0 +1,55 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO systemd/systemd
+  REF "v${VERSION}"
+  SHA512 3bbc431a292ab590b70d3b490a528f71d30ccf478ddfa66d1c210f40c260ef49ac30651c19f2d073acf38d68398a4a6fbf95391f0e3ea0333d94b9d4e81d514f
+)
+
+vcpkg_configure_meson(
+  SOURCE_PATH "${SOURCE_PATH}"
+  OPTIONS
+    -Dstatic-libsystemd=true
+    -Daudit=false
+  OPTIONS_DEBUG
+    -Drootprefix=${CURRENT_PACKAGES_DIR}/debug
+    -Dpkgconfiglibdir={CURRENT_PACKAGES_DIR}/debug
+  OPTIONS_RELEASE
+    -Drootprefix=${CURRENT_PACKAGES_DIR}
+    -Dpkgconfiglibdir={CURRENT_PACKAGES_DIR}
+)
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+  vcpkg_build_ninja(
+    TARGETS libsystemd.a devel
+  )
+else()
+  vcpkg_build_ninja(
+    TARGETS libsystemd devel
+  )
+endif()
+
+file(INSTALL ${SOURCE_PATH}/src/systemd DESTINATION ${CURRENT_PACKAGES_DIR}/include FILES_MATCHING PATTERN "*.h")
```

Comment:
`ninja install` tries to build all targets that are configured, so our best bet would be to only configure build target libsystemd than this could be achieved with `ninja install`